Update detaching-a-fork.md

fgnievinski opened the pull request:
link to option for detaching/extracting a fork

<!--
Thank you for contributing to this project! You must fill out the information below before we can review this pull request. By explaining why you're making a change (or linking to an issue) and what changes you've made, we can triage your pull request to the best possible team for review.
-->

### Why: currently fork detachment/extraction is not documented.

Closes: none

<!-- If there's an existing issue for your change, please link to it above.
If there's _not_ an existing issue, please open one first to make it more likely that this update will be accepted: https://github.com/github/docs/issues/new/choose. -->

### What's being changed (if available, include any code snippets, screenshots, or gifs):

only added a link for further information:
![image](https://github.com/github/docs/assets/5018500/a93c9175-3605-4a99-b8d1-420db99fd61d)

<!-- Let us know what you are changing. Share anything that could provide the most context.
If you made changes to the `content` directory, a table will populate in a comment below with links to the preview and current production articles. -->

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

fgnievinski committed:
Update detaching-a-fork.md

link to option for detaching/extracting a fork

nguyenalex836 commented:
@fgnievinski Thanks for opening a PR! ✨ Please fill out the [pull request template](https://raw.githubusercontent.com/github/docs/main/.github/PULL_REQUEST_TEMPLATE.md) in the top comment in this pull request, then we'll get this up for review!

fgnievinski commented:
top comment updated to fill out the PR template.

nguyenalex836 commented:
@fgnievinski Thank you for filling out the template! I'll get this triaged for review ✨

mchammer01 commented on content/pull-requests/collaborating-with-pull-requests/working-with-forks/detaching-a-fork.md:
````diff
@@ -68,3 +68,6 @@ You can delete a fork and recreate the same repository, without the connection t
    rm -rf FORK-NAME.git
    ```
+
+## See also
````
Unsure as to whether we want to add the link to Support here, but fixing the linter bug so that there's no test failure.

```suggestion
## See also

```

mchammer01 committed:
Update content/pull-requests/collaborating-with-pull-requests/working-with-forks/detaching-a-fork.md

mchammer01 committed:
Merge branch 'main' into patch-1

mchammer01 commented on content/pull-requests/collaborating-with-pull-requests/working-with-forks/detaching-a-fork.md:
```diff
@@ -68,3 +68,7 @@ You can delete a fork and recreate the same repository, without the connection t
+
+## See also
+
+* [Detach or extract a fork](https://support.github.com/request/fork)
```
Another linter fix
```suggestion
- [Detach or extract a fork](https://support.github.com/request/fork)
```

mchammer01 committed:
Update content/pull-requests/collaborating-with-pull-requests/working-with-forks/detaching-a-fork.md

nguyenalex836 committed:
Merge branch 'main' into patch-1

nguyenalex836 committed:
Update detaching-a-fork.md

Updates to adhere more closely to our style guide - https://docs.github.com/en/contributing/style-guide-and-content-model/style-guide#links

nguyenalex836 commented:
@fgnievinski Thanks so much for opening a PR with this update! ✨ I've modified the changes slightly to more closely adhere to our [style guide](https://docs.github.com/en/contributing/style-guide-and-content-model/style-guide#links).

I'll update the branch and get this merged once tests are passing 🍏

nguyenalex836 committed:
Merge branch 'main' into patch-1
